tool_for_bioinformatics.py

```diff
@@ -0,0 +1,153 @@
+            the last  variable must be the function (str) you want to execute.
+            The amino acid sequence can consist of both uppercase and lowercase letters.
+        Input example:
+            amino_acid_tools('PLPKVEL','VDviRIkLQ','PPDFGKT','folding')
+        Function:
+            molecular_weight: calculates molecular weight of the amino acid chain
+            three_letter_code: converts single letter translations to three letter translations
+            length: counts the number of amino acids in the given sequence
+            folding: counts the number of amino acids characteristic separately for alpha helixes and beta sheets,
+                    and gives out what will be the structure of the protein more
+            seq_charge: evaluates the overall charge of the aminoacid chain in neutral aqueous solution (pH = 7)
+
+        Returns:
+            If one sequence is supplied, a string with the result is returned.
+            If several are submitted, a list of strings is returned.
+            Depending on the function performed, the following returns will occur:
+                molecular_weight (int) or (list): amino acid sequence molecular weight number or list of numbers
+                three_letter_code (str) or (list): translated sequence from one-letter in three-letter code
+                length (int) or (list): integer number of amino acid residues
+                folding (str) or (list): 'alpha_helix', if there are more alpha helices
+                                        'beta_sheet', if there are more beta sheets
+                                        'equally', if the probability of alpha spirals and beta sheets are the same
+                seq_charge(str) or (list): "positive", "negative" or "neutral"
+            If seq contains symbols differ from IUPAC 1letter code for 22 proteinogenic amino acids,
+            the result for this sequence will be 'unexpected symbols in sequence' regardless of function
+            If action is not in the function the message 'unexpected action' will occur
+    """
+    *seqs, function = args
+    d_of_functions = {'molecular_weight': aa.molecular_weight,
+                      'three_letter_code': aa.three_letter_code,
+                      'length': aa.length,
+                      'folding': aa.folding,
+                      'seq_charge': aa.seq_charge}
+    answer = []
+    if function not in d_of_functions.keys():
+        raise ValueError("unexpected action")
+    for sequence in seqs:
+        if not set(sequence).issubset(AMINO_ACIDS):
+            answer.append('unexpected symbols in sequence')
+        else:
+            answer.append(d_of_functions[function](sequence))
+    if len(answer) == 1:
+        return answer[0]
+    else:
+        return answer
+
+
+def fastq_filtration(input_fastq, gc_bounds=(0, 100), length_bounds=(0, 2 ** 32), quality_treshold=0, output_fastq=''):

```

Comment:
1) Функции глаголами
2) Пропущенные значения по умолчанию - None
```suggestion
def run_fastq_filtration(input_fastq, gc_bounds=(0, 100), length_bounds=(0, 2 ** 32), quality_treshold=0, output_fastq=None):
```
или
```suggestion
def filter_fastq(input_fastq, gc_bounds=(0, 100), length_bounds=(0, 2 ** 32), quality_treshold=0, output_fastq=None):
```

Reply:
А еще в ТЗ первый аргумент это `input_path`, ну и + в слове treshold опечатка (`quality_threshold`) :)